### PR TITLE
[https://nvbugs/6422432][fix] Unwaive testcase

### PR DESCRIPTION
**Pull request by YihuiLu512**
## Dev Engineer Review

- Updated `tests/integration/test_lists/waives.txt`.
- Removed obsolete DeepSeek R1 latency and DeepSeekV3Lite NVFP4 waivers.
- Added waivers for GLM52 NVFP4, Qwen softmax attention, LTX2 CUDA-graph TRTLLM, DGX B200 performance alignment, Exaone attention backends, Qwen3 LoRA, standalone packaging, Kimi K3 generation performance, and related performance scenarios.
- No API, performance, or error-handling code changed.
- The available diff output did not show the complete entry-level changes. Verify waiver format, bug references, test paths, scope, and duplicate entries.

## QA Engineer Review

- Modified file: `tests/integration/test_lists/waives.txt`.
- Added waivers for the GLM52, Qwen, LTX2, DGX B200, Exaone, Qwen3, standalone packaging, Kimi K3, and related performance scenarios.
- Removed DeepSeek R1 latency and DeepSeekV3Lite NVFP4 waivers.
- No `test-db/` or `qa/` files were modified.
- Verdict: needs follow-up because CBTS coverage data is unavailable.

## Test Coverage
```
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2"
```

## PR Checklist

Please review the following before submitting your PR:
- PR description clearly explains what and why. If using CodeRabbit's summary, please make sure it makes sense.
- PR Follows [TRT-LLM CODING GUIDELINES](https://github.com/NVIDIA/TensorRT-LLM/blob/main/CODING_GUIDELINES.md) to the best of your knowledge.
- Test cases are provided for new code paths (see [test instructions](https://github.com/NVIDIA/TensorRT-LLM/tree/main/tests#1-how-does-the-ci-work))
- If PR introduces API changes, an appropriate PR label is added - either `api-compatible` or `api-breaking`. For `api-breaking`, include `BREAKING` in the PR title.
- Any new dependencies have been scanned for license and vulnerabilities
- [CODEOWNERS](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/CODEOWNERS) updated if ownership changes
- Documentation updated as needed
- Update [tava architecture diagram](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/tava_architecture_diagram.md) if there is a significant design change in PR.
- The reviewers assigned automatically/manually are appropriate for the PR.

- [x] Please check this after reviewing the above items as appropriate for this PR.

## GitHub Bot Help

To see a list of available CI bot commands, please comment `/bot help`.

### CODE REVIEW

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2"

**Review by BowenFu — APPROVED** [review]

**Comment by YihuiLu512** [comment]
@QiJune Please add the following label so that I can verify whether the bug is reproducible: `ci: post-merge approved`.

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --disable-reuse-test --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --disable-reuse-test --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-1, RTXPro6000D-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --extra-stage "DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast

**Comment by YihuiLu512** [comment]
/bot run --stage-list "GB200-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-1, DGX_B300-4_GPUs-PyTorch-Post-Merge-2" --disable-fail-fast
